skimage/util/tests/test_compare.py

```diff
@@ -20,8 +19,24 @@ def test_compare_images_diff():
     img2[3:8, 0:8] = 255
     expected_result = np.zeros_like(img1, dtype=np.float64)
     expected_result[3:8, 0:3] = 1
-    result = compare_images(img1, img2, method='diff')
-    assert_array_equal(result, expected_result)
+    with pytest.warns(FutureWarning):
+        result = compare_images(img1, img2, method='diff')
+    np.testing.assert_array_equal(result, expected_result)
+
+
+def test_compare_images_replaced_param():
+    img1 = np.zeros((10, 10), dtype=np.uint8)
+    img1[3:8, 3:8] = 255
+    img2 = np.zeros_like(img1)
+    img2[3:8, 0:8] = 255
+    with pytest.warns(FutureWarning):
```

Comment:
Might at least use `match=".*image[01]` here and elsewhere.

Reply:
I'm not sure... I removed the case where input images are passed as kwargs following the new API (e0d5929d4e321c1ae), so we're not quite cycling over all image0-image1 combinations when expecting a warning. Or maybe my command of regular expressions is not up to the mark :wink: